package.json

```diff
@@ -97,8 +105,8 @@
     "ts-jest": "^26.1.0",
     "tslint": "^6.1.2",
     "tslint-config-standard": "^9.0.0",
-    "typedoc": "^0.16.0",
-    "typescript": "~3.6.5"
+    "typedoc": "^0.19.0",
+    "typescript": "^3.9.7"
```

Comment:
I was getting warnings from jest that it had not been tested with the version of typescript in use. It recommended a version between 3.8 and <4.0. But we could bump jest too and go to something >=4.0? 
The gateways are building OK ... but we should move forward soon.

Reply:
Yeah, if jest can go newer then I think update that too. We may as well update typescript to latest all in one step, rather than having to come back to it later

Reply:
Reason was jest moaning that it needed a version >3.8 and <=4.0. Plus a (failed) attempt to resolve typedoc external dependency issue.
Consuming gateways seem to be happy to build like this, but we should be generally updating anyway.

Reply:
As of https://github.com/nrkno/tv-automation-server-core-integration/pull/34/commits/0a7b14bd0700921ed2e693a86f79568b41dccb7d, Typescript is ^4.0.3, tslib ^2.0.1 and jest is not moaning.  

Will test with media manager and see how it goes.